tests/sentry/replays/tasks/test_delete_replays_bulk.py

```diff
@@ -180,6 +180,24 @@ def test_run_bulk_replay_delete_job_chained_runs(self):
         assert self.job.status == "completed"
         assert self.job.offset == 2
 
+    def test_run_bulk_replay_delete_job_already_failed(self):
+        t1 = datetime.datetime.now() - datetime.timedelta(seconds=10)
+        replay_id1 = uuid.uuid4().hex
+        self.store_replays(
+            mock_replay(t1, self.project.id, replay_id1, segment_id=0, environment="prod")
+        )
+
+        self.job.status = DeletionJobStatus.FAILED
+        self.job.save()
+
+        with TaskRunner():
+            run_bulk_replay_delete_job.delay(self.job.id, offset=0, limit=0)
+
+        # Runs were chained.
```

Comment:
```suggestion
        # Does not re-run.
```
Wdym by runs were chained?